src/modules/todo/types/Todo.interface.ts

```diff
@@ -0,0 +1,6 @@
+ interface TodoItem {
```

Comment:
Better to use type instead of interface. The reason behind this is that TodoItem is self-sufficient element and it's highliy unlickely that you will exted from it.